Makefile

```diff
@@ -22,7 +22,7 @@ build-openshift-ci-test-bin:
 	./hack/setup_env.sh
 
 lint:
-	golangci-lint run -v
+	golangci-lint run -v --timeout=5m
```

Comment:
Not that I have something against adding this timeout, but I wonder: why is it needed here and not [in assisted-service?](https://github.com/openshift/assisted-service/blob/b253ac87973b987216df5940c9857e7d4815a65c/Makefile#L193-L194)

Reply:
Ack. Merged it.